server/index.html

```diff
@@ -2,15 +2,16 @@
 <html>
 
 <body>
+    <h4>Create User</h4>
     <form action="/user" method="POST">
         <label for="firstName">First name:</label><br>
         <input type="text" id="firstName" name="firstName" value="David"><br>
 
         <label for="lastName">Last name:</label><br>
         <input type="text" id="lastName" name="lastName" value="Michael"><br>
 
-        <label for="birthday">Birthday:</label><br>
-        <input type="text" id="birthday" name="birthday" value="1/1/1970"><br>
+        <label for="dateOfBirth">Birthday:</label><br>
```

Comment:
What are all the hard coded values in this file?

Reply:
Is this just your test file? if so do we need it or can it be removed?

Reply:
test file only